changelog/july2023/2023-07-18-public-gateways-changed-the-public-gateway-is.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Public Gateway is changing
+status: changed
+author:
+    fullname: 'Join the #public-gateway channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-07-18
+category: network
+product: public-gateways
+---
+
+With the release of VPC in GA, Public Gateway no longer supports the DHCP server.
+Public Gateway now focuses on its primary functions such as:
+- Internet access: allowing external traffic to access the internal resources of a Private Network
+- NAT-Gateway: distributing requests to the right IP addresses
+- SSH Bastion: allowing and monitoring SSH connections
+- Local DNS server: customizing resources name on the Private Network
+
```

Comment:
See our [dedicated documentation](https://www.scaleway.com/en/docs/network/vpc/reference-content/vpc-migration/) for full details.